app/src/main/kotlin/com/wire/android/util/permission/CaptureVideoRequestFlow.kt

```diff
@@ -8,30 +8,38 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.remember
 import androidx.compose.ui.platform.LocalContext
 
+/**
+ * Flow that will launch the camera to record a video.
+ * This will handle the permissions request in case there is no permission granted for the camera.
+ *
+ * @param onVideoRecorded action that will be executed for Camera's [ActivityResultContract]
+ * @param onPermissionDenied action to be executed when the permissions is denied
+ * @param targetVideoFileUri target file where the media will be stored
+ */
 @Composable
 fun rememberCaptureVideoFlow(
-    shouldPersistUri: (Boolean) -> Unit,
+    onVideoRecorded: (Boolean) -> Unit,
     onPermissionDenied: () -> Unit,
-    onVideoCapturedUri: Uri
+    targetVideoFileUri: Uri
 ): UseCameraRequestFlow {
     val context = LocalContext.current
 
     val captureVideoLauncher: ManagedActivityResultLauncher<Uri, Boolean> = rememberLauncherForActivityResult(
         ActivityResultContracts.CaptureVideo()
     ) { hasCapturedVideo ->
-        shouldPersistUri(hasCapturedVideo)
+        onVideoRecorded(hasCapturedVideo)
     }
 
     val requestPermissionLauncher: ManagedActivityResultLauncher<String, Boolean> =
```

Comment:
could we rename this one to requestVideoPermissionLauncher

Reply:
Done! :D